coalib/misc/Constants.py

```diff
@@ -109,7 +110,11 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+if os.path.isfile(os.path.join(coalib_root, 'system_coafile')):
+    system_coafile = os.path.join(coalib_root, 'system_coafile')
+else:
+    system_coafile = os.path.join(coalib_root, 'default_coafile')
+    logging.warning('deprecated')
```

Comment:
need to use logging.getLogger before this and call the warning on the logger object it returns. This makes sure that the config is applied correctly. Also, the deprecation message needs to be more descriptive